python/private/proto/py_proto_library.bzl

```diff
@@ -75,12 +75,21 @@ def _py_proto_aspect_impl(target, ctx):
             name_mapper = lambda name: name.replace("-", "_").replace(".", "/"),
         )
 
+        proto_root = proto_info.proto_source_root
```

Comment:
nit: add a comment that explains this is to handle when multiple repositories are involved and virtual imports. It's easy to forget when different repos are involved and sort of a pain to test it, so lets at least comment it.

Reply:
Done.